<ui version="4.0" stdsetdef="1" >
  <author></author>
  <class>TGo4ScaleOptions</class>
  <widget class="QGo4Widget" name="TGo4ScaleOptions" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>367</width>
        <height>34</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Histogram Scale Options</string>
    </property>
    <property name="windowIcon">
      <iconset>:/icons/go4logo2_big.png</iconset>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <property name="spacing" >
        <number>0</number>
      </property>
      <item row="0" column="0" >
        <widget class="QSpinBox" name="ExpandFactor" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="maximum" >
            <number>100</number>
          </property>
          <property name="singleStep" >
            <number>5</number>
          </property>
          <property name="value" >
            <number>10</number>
          </property>
        </widget>
      </item>
      <item row="0" column="1" >
        <widget class="QLabel" name="TextLabel1" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="text" >
            <string>%</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="0" column="2" >
        <widget class="QToolButton" name="ExpandX" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="maximumSize" >
            <size>
              <width>24</width>
              <height>30</height>
            </size>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <pixmap>:/icons/zoomxin.png</pixmap>
          </property>
          <property name="autoRepeat" >
            <bool>true</bool>
          </property>
          <property name="icon" >
            <iconset>:/icons/zoomxin.png</iconset>
          </property>
          <property name="shortcut">
             <string>Ctrl+Left</string>
          </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
             <string>Zoom X axis. Keyboard: Ctrl+Left</string>
          </property>
        </widget>
      </item>
      <item row="0" column="3" >
        <widget class="QToolButton" name="ShiftL" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="maximumSize" >
            <size>
              <width>24</width>
              <height>30</height>
            </size>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <pixmap>:/icons/shiftleft.png</pixmap>
          </property>
          <property name="autoRepeat" >
            <bool>true</bool>
          </property>
          <property name="icon" >
            <iconset>:/icons/shiftleft.png</iconset>
          </property>
            <property name="shortcut">
                <string>Left</string>
            </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
                <string>Shift graph down X. Keyboard:(Left)</string>
          </property>
        </widget>
      </item>
      <item row="0" column="4" >
        <widget class="QToolButton" name="ShiftR" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="maximumSize" >
            <size>
              <width>24</width>
              <height>30</height>
            </size>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <pixmap>:/icons/shiftright.png</pixmap>
          </property>
          <property name="autoRepeat" >
            <bool>true</bool>
          </property>
          <property name="icon" >
            <iconset>:/icons/shiftright.png</iconset>
          </property>
          <property name="shortcut">
                <string>Right</string>
            </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
                <string>Shift graph up X. Keyboard:(Right)</string>
          </property>
        </widget>
      </item>
      <item row="0" column="5" >
        <widget class="QToolButton" name="UExpandX" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="maximumSize" >
            <size>
              <width>24</width>
              <height>30</height>
            </size>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <pixmap>:/icons/zoomxout.png</pixmap>
          </property>
          <property name="autoRepeat" >
            <bool>true</bool>
          </property>
          <property name="icon" >
            <iconset>:/icons/zoomxout.png</iconset>
          </property>
           <property name="shortcut">
                <string>Ctrl+Right</string>
            </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
                <string>Unzoom X axis. Keyboard:(Ctrl+Right)</string>
          </property>
        </widget>
      </item>
      <item row="0" column="6" >
        <widget class="QToolButton" name="ExpandY" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="maximumSize" >
            <size>
              <width>24</width>
              <height>30</height>
            </size>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <pixmap>:/icons/zoomyin.png</pixmap>
          </property>
          <property name="autoRepeat" >
            <bool>true</bool>
          </property>
          <property name="icon" >
            <iconset>:/icons/zoomyin.png</iconset>
          </property>
          <property name="shortcut">
                <string>Down</string>
          </property>          
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
                <string>Zoom Y axis. Keyboard:(Down)</string>
          </property>
        </widget>
      </item>
      <item row="0" column="7" >
        <widget class="QToolButton" name="ShiftDown" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="maximumSize" >
            <size>
              <width>24</width>
              <height>30</height>
            </size>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <pixmap>:/icons/shiftdown.png</pixmap>
          </property>
          <property name="autoRepeat" >
            <bool>true</bool>
          </property>
          <property name="icon" >
            <iconset>:/icons/shiftdown.png</iconset>
          </property>
          <property name="shortcut">
                <string>Shift+Down</string>
          </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
                <string>Shift graph down Y. Keyboard:(Shift+Down)</string>
          </property>
        </widget>
      </item>
      <item row="0" column="8" >
        <widget class="QToolButton" name="ShiftUp" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="maximumSize" >
            <size>
              <width>24</width>
              <height>30</height>
            </size>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <pixmap>:/icons/shiftup.png</pixmap>
          </property>
          <property name="autoRepeat" >
            <bool>true</bool>
          </property>
          <property name="icon" >
            <iconset>:/icons/shiftup.png</iconset>
          </property>
          <property name="shortcut">
                <string>Shift+Up</string>
          </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
                <string>Shift graph up Y. Keyboard:(Shift+Up)</string>
          </property>
        </widget>
      </item>
      <item row="0" column="9" >
        <widget class="QToolButton" name="UExpandY" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="maximumSize" >
            <size>
              <width>24</width>
              <height>30</height>
            </size>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <pixmap>:/icons/zoomyout.png</pixmap>
          </property>
          <property name="autoRepeat" >
            <bool>true</bool>
          </property>
          <property name="icon" >
            <iconset>:/icons/zoomyout.png</iconset>
          </property>
          <property name="shortcut">
               <string>Up</string>
          </property>          
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
             <string>Unzoom Y axis. Keyboard:(Up)</string>
          </property>
        </widget>
      </item>
      <item row="0" column="10" >
        <widget class="QToolButton" name="ExpandZ" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="maximumSize" >
            <size>
              <width>24</width>
              <height>30</height>
            </size>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <pixmap>:/icons/zoomzin.png</pixmap>
          </property>
          <property name="autoRepeat" >
            <bool>true</bool>
          </property>
          <property name="icon" >
            <iconset>:/icons/zoomzin.png</iconset>
          </property>
          <property name="shortcut">
                <string>Ctrl+End</string>
           </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Zoom Z axis. Keyboard: (Ctrl+End)</string>
          </property>
        </widget>
      </item>
      <item row="0" column="11" >
        <widget class="QToolButton" name="ShiftDownZ" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="maximumSize" >
            <size>
              <width>24</width>
              <height>30</height>
            </size>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <pixmap>:/icons/shiftzdown.png</pixmap>
          </property>
          <property name="autoRepeat" >
            <bool>true</bool>
          </property>
          <property name="icon" >
            <iconset>:/icons/shiftzdown.png</iconset>
          </property>
            <property name="shortcut">
                <string>Shift+End</string>
           </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Shift graph down Z. Keyboard:(Shift+End)</string>
          </property>
        </widget>
      </item>
      <item row="0" column="12" >
        <widget class="QToolButton" name="ShiftUpZ" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="maximumSize" >
            <size>
              <width>24</width>
              <height>30</height>
            </size>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <pixmap>:/icons/shiftzup.png</pixmap>
          </property>
           <property name="shortcut">
                <string>Shift+Home</string>
           </property>
          <property name="autoRepeat" >
            <bool>true</bool>
          </property>
          <property name="icon" >
            <iconset>:/icons/shiftzup.png</iconset>
          </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Shift graph up Z. Keyboard:(Shift+Home)</string>
          </property>
        </widget>
      </item>
      <item row="0" column="13" >
        <widget class="QToolButton" name="UExpandZ" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="maximumSize" >
            <size>
              <width>24</width>
              <height>30</height>
            </size>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <pixmap>:/icons/zoomzout.png</pixmap>
          </property>
          <property name="autoRepeat" >
            <bool>true</bool>
          </property>
          <property name="icon" >
            <iconset>:/icons/zoomzout.png</iconset>
          </property>
           <property name="shortcut">
                <string>Ctrl+Home</string>
           </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Unzoom Z axis. Keyboard:(Ctrl+Home)</string>
          </property>
        </widget>
      </item>
      <item row="0" column="14" >
        <widget class="QToolButton" name="resize_b" >
          <property name="minimumSize" >
            <size>
              <width>0</width>
              <height>0</height>
            </size>
          </property>
          <property name="maximumSize" >
            <size>
              <width>30</width>
              <height>30</height>
            </size>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <pixmap>:/icons/zoomoff.png</pixmap>
          </property>
          <property name="icon" >
            <iconset>:/icons/zoomoff.png</iconset>
          </property>
          <property name="shortcut">
            <string>Ctrl+-</string>
          </property>
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Unzoom to full ranges. Keyboard: Ctrl+Minus</string>
          </property>
        </widget>
      </item>
      <item row="0" column="15" >
        <widget class="QToolButton" name="SetScale" >
          <property name="maximumSize" >
            <size>
              <width>30</width>
              <height>30</height>
            </size>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <pixmap>:/icons/zoomlim.png</pixmap>
          </property>
          <property name="icon" >
            <iconset>:/icons/zoomlim.png</iconset>
          </property>
          <property name="shortcut">
            <string>Ctrl++</string>
          </property> 
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
             <string>Open zoom and autoscale window. Keyboard: Ctrl+Plus</string>
          </property>
        </widget>
      </item>
      <item row="0" column="16" >
        <widget class="QToolButton" name="AutoZoomBtn" >
          <property name="maximumSize" >
            <size>
              <width>30</width>
              <height>30</height>
            </size>
          </property>
          <property name="text" >
            <string/>
          </property>
          <property name="icon" >
            <pixmap>:/icons/zoomauto.png</pixmap>
          </property>
          <property name="icon" >
            <iconset>:/icons/zoomauto.png</iconset>
          </property>
          <property name="shortcut">
            <string>Ctrl+*</string>
          </property> 
          <property name="autoRaise" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
             <string>Automatically zoom histogram to filled region. Keyboard: Ctrl+*</string>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="2" margin="2" />
  <includes>
    <include location="local">QGo4Widget.h</include>
  </includes>
  <connections>
    <connection>
      <sender>resize_b</sender>
      <signal>clicked()</signal>
      <receiver>TGo4ScaleOptions</receiver>
      <slot>resizeall()</slot>
    </connection>
    <connection>
      <sender>SetScale</sender>
      <signal>clicked()</signal>
      <receiver>TGo4ScaleOptions</receiver>
      <slot>SetScaleValues()</slot>
    </connection>
    <connection>
      <sender>AutoZoomBtn</sender>
      <signal>clicked()</signal>
      <receiver>TGo4ScaleOptions</receiver>
      <slot>AutoZoomScales()</slot>
    </connection>
    <connection>
      <sender>UExpandX</sender>
      <signal>clicked()</signal>
      <receiver>TGo4ScaleOptions</receiver>
      <slot>SlotUExpandX()</slot>
    </connection>
    <connection>
      <sender>ShiftL</sender>
      <signal>clicked()</signal>
      <receiver>TGo4ScaleOptions</receiver>
      <slot>SlotShiftL()</slot>
    </connection>
    <connection>
      <sender>ShiftR</sender>
      <signal>clicked()</signal>
      <receiver>TGo4ScaleOptions</receiver>
      <slot>SlotShiftR()</slot>
    </connection>
    <connection>
      <sender>ExpandX</sender>
      <signal>clicked()</signal>
      <receiver>TGo4ScaleOptions</receiver>
      <slot>SlotExpandX()</slot>
    </connection>
    <connection>
      <sender>ShiftUp</sender>
      <signal>clicked()</signal>
      <receiver>TGo4ScaleOptions</receiver>
      <slot>SlotShiftU()</slot>
    </connection>
    <connection>
      <sender>UExpandY</sender>
      <signal>clicked()</signal>
      <receiver>TGo4ScaleOptions</receiver>
      <slot>SlotUExpandY()</slot>
    </connection>
    <connection>
      <sender>ExpandY</sender>
      <signal>clicked()</signal>
      <receiver>TGo4ScaleOptions</receiver>
      <slot>SlotExpandY()</slot>
    </connection>
    <connection>
      <sender>ShiftDown</sender>
      <signal>clicked()</signal>
      <receiver>TGo4ScaleOptions</receiver>
      <slot>SlotShiftD()</slot>
    </connection>
    <connection>
      <sender>ShiftUpZ</sender>
      <signal>clicked()</signal>
      <receiver>TGo4ScaleOptions</receiver>
      <slot>SlotShiftZU()</slot>
    </connection>
    <connection>
      <sender>UExpandZ</sender>
      <signal>clicked()</signal>
      <receiver>TGo4ScaleOptions</receiver>
      <slot>SlotUExpandZ()</slot>
    </connection>
    <connection>
      <sender>ExpandZ</sender>
      <signal>clicked()</signal>
      <receiver>TGo4ScaleOptions</receiver>
      <slot>SlotExpandZ()</slot>
    </connection>
    <connection>
      <sender>ShiftDownZ</sender>
      <signal>clicked()</signal>
      <receiver>TGo4ScaleOptions</receiver>
      <slot>SlotShiftZD()</slot>
    </connection>
  </connections>
</ui>
